### PR TITLE
migration: validate vocabulary values

### PR DESCRIPTION
**Pull request by KonstantinaStoikou**
Closes https://github.com/orgs/inveniosoftware/projects/19#card-57011896

### CODE REVIEW

**Review by kprzerwa — COMMENTED** [review]
I would prefer to verify it on the level of the rules, not the whole record, because it adds complexity with the mapping @ntarocco WDYT?

File: cds_ils/cli.py [hunk 1]
```diff
@@ -229,6 +229,8 @@ def run_command(command, catch_exceptions=False, verbose=True):
     run_command("vocabulary index opendefinition opendefinition --force")
     # languages
     run_command("vocabulary index languages --force")
+    # countries
+    run_command("vocabulary index languages --force")
```
Comment (KonstantinaStoikou) [inline cds_ils/cli.py:233]: why do we need to run the same command twice?

Comment (ntarocco) [reply cds_ils/cli.py]: Because it should be `countries` instead! Thank you for spotting this, fixed! :) 👍🏼

File: cds_ils/migrator/cli.py [hunk 2]
```diff
@@ -65,6 +68,7 @@ def migration():
         "acq-order": records_formatter,
         "eitem": eitems_formatter,
         "relation": records_formatter,
+        "vocabularie": records_formatter,
```
Comment (kprzerwa) [inline cds_ils/migrator/cli.py:71]: uff sorry about the `s`

**Review by kprzerwa — APPROVED** [review]
LGTM!
If you don't mind I will check it out before merging and I will try to run the migration on it to make sure that the migration scripts will run correctly

**Review by jrcastro2 — APPROVED** [review]

**Pull request merged**